static/js/settings_account.js

```diff
@@ -216,6 +212,22 @@ exports.add_custom_profile_fields_to_settings = function () {
     exports.initialize_custom_date_type_fields(element_id);
 };
 
+function full_name_updated_successfully() {
+    overlays.close_modal('change_full_name_modal');
+}
```

Comment:
I think it'd be better to put these functions inline in the `opts` definitions, since they're so small and only used in one place.